.Documentation/articles/pro/03-desktop-mode.md

```diff
@@ -0,0 +1,63 @@
+# Desktop Mode
+
+The Desktop Mode is part of our Innoactive Creator PRO offer and allows you to run your training applications without the need of a VR headset. You can perform the training using keyboard and mouse on a regular desktop PC. The training course created in Innoactive Creator does not need to be adapted to run in desktop mode. Just build your training for both VR and non-VR and you can switch even during runtime between a training experience in VR and a training experience using keyboard and mouse.
+
+## Setup in Unity
+
+Define if you want to start in Desktop Mode or in VR when launching your training application. You have to configure two game objects in your training scene.
+
+First the _[COURSE_CONTROLLER]_:
+
+[![Course Controller](../images/pro/03-course-controller.png "Course controller settings.")](../images/pro/03-course-controller.png)
+
+When using the _Desktop Mode_ you have to make sure that _Desktop_ is chosen in the _Course Controller Setup_. For VR (and the possibility to switch to Desktop during runtime) choose _Advanced_. You do not have to make any changes on any other component on the _Course Controller_ game object, everything will be automatically configured.
+
+Second the _[INTERACTION_RIG_LOADER]_:
+
+[![Interaction Rig](../images/pro/03-rig-loader.png "Interaction rig settings.")](../images/pro/03-rig-loader.png)
+
+Moving the _Novice Desktop Mode_ rig to the top will ensure that the desktop mode (camera and interactions) will be loaded. Choosing any XR rig (_XR Rig_ is recommended) will load a VR configured camera rig and controllers as soon as the application starts.
+
+_Note:_ For testing in Unity you can switch between settings, but before you build your final application make sure you have the desired starting configuration set in both objects.
+
+## How to use the _novice_ Desktop Mode
+
+The novice Desktop mode is targeted on 'conservative' users, not used to playing computer games or to any advanced interaction with keyboard and mouse. We intentionally reduced the freedom of control to reduce complexity and increased the usability by simplifying the interaction. We will soon release the _advanced_ desktop mode providing more control. 
```

Comment:
`We will soon release the _advanced_ desktop mode providing more control.`

Either we update this as soon as the advance mode is released or we just never publish it, this is not a newsletter.

Reply:
You are right. Removed that line but kept the section on the bottom of the page where we say that an advanced mode will come. is that fine with you?

Reply:
That works for me 👍